cpp/modmesh/buffer/BufferExpander.hpp

```diff
@@ -31,10 +31,10 @@
 #include <modmesh/base.hpp>
 #include <modmesh/buffer/ConcreteBuffer.hpp>
 
-#include <stdexcept>
-#include <memory>
 #include <algorithm>
+#include <memory>
 #include <sstream>
+#include <stdexcept>
```

Comment:
I don't like clang-format to reorder the include file.  If you have time please find a switch to turn off the reordering (later).

Reply:
Because the order has meaning.

Reply:
@yungyuc  to prevent reordering, you can add a new empty line, then clang-format will not do reordering.

Reply:
That is ugly